1-exercises/A-accessing-values/exercise3.js

```diff
@@ -22,11 +22,20 @@ let basketballTeam = {
 
 // write code here
 
+let  arrayOfNames= basketballTeam["topPlayers"].sort((a,b) => a.localeCompare(b))
```

Comment:
as you are sorting strings alphabetically here you can just use `.sort()` so `basketballTeam.topPlayers.sort()` :) and then some kind of loop to loop over each player & console.log their name